flink-runtime/src/main/java/org/apache/flink/runtime/state/TaskExecutorFileMergingManager.java

```diff
@@ -74,8 +79,9 @@ public TaskExecutorFileMergingManager() {
      * Initialize file merging snapshot manager for each job according configurations when {@link
      * org.apache.flink.runtime.taskexecutor.TaskExecutor#submitTask}.
      */
-    public @Nullable FileMergingSnapshotManager fileMergingSnapshotManagerForJob(
+    public @Nullable FileMergingSnapshotManager fileMergingSnapshotManagerForTask(
```

Comment:
How about renaming it to `fileMergingSnapshotManagerForAttempt`?

Reply:
I'd prefer the `Task`, since this manager will be used to create a `TaskStateManager` which will be given to a `Task`. The `Task` is more in line with the caller’s cognition.